cssom-1/Overview.bs

```diff
@@ -2034,6 +2038,40 @@ Note: The serialization of an empty CSS declaration block is the empty string.
 Note: The serialization of a non-empty CSS declaration block does not include any surrounding whitespace, i.e., no whitespace appears
 before the first property name and no whitespace appears after the final semicolon delimiter that follows the last property value.
 
+A <a>CSS declaration block</a> has these <a>attribute change steps</a> for its <a for="CSSStyleDeclaration">owner node</a>:
+
+<ol>
+ <li>If the <a for="CSSStyleDeclaration">readonly flag</a> is set, terminate these steps.
+ <li>If the <a for="CSSStyleDeclaration">updating flag</a> is set, terminate these steps.
+ <li>If <var>localName</var> is not "<code>style</code>", or <var>namespace</var> is not null, terminate these steps.
+ <li>If <var>value</var> is null, empty the <a for="CSSStyleDeclaration">declarations</a>.
+ <li>Otherwise, let the <a for="CSSStyleDeclaration">declarations</a> be the result of <a>parse a CSS declaration block</a>
+ from a string <var>value</var>.
+</ol>
+
+When a <a>CSS declaration block</a> object is created, then:
```

Comment:
@upsuper Why is this timing correct? Especially since nothing specifies when these objects are actually created, last I checked...

Reply:
As far as it is returned from `Element.style`, does it matter when the objects are actually created? Browser needs to create it at some point, but before script gets it somehow, whether it exists doesn't really matter, does it?

Reply:
It matters because the base URI is different at different points in time, depending on what the node document of the element is.  So depending on exactly when the creation/parsing happen you will get different behavior for relative URIs in property values....

Maybe that doesn't affect what CSSOM sees, just the actual styling behavior.

Reply:
Hmm... okay. Sounds like something worth raising for css-style-attr I guess... The interaction between the two specs would be fun...